qiskit/quantum_info/synthesis/one_qubit_decompose.py

```diff
@@ -281,91 +281,112 @@ def _params_u1x(mat):
         return theta, phi, lam, phase - 0.5 * (theta + phi + lam)
 
     @staticmethod
-    def _circuit_zyz(theta, phi, lam, phase, simplify=True, atol=DEFAULT_ATOL):
+    def _circuit_kak(
+        theta,
+        phi,
+        lam,
+        phase,
+        simplify=True,
+        atol=DEFAULT_ATOL,
+        allow_non_canonical=True,
+        k_gate=RZGate,
+        a_gate=RYGate,
+    ):
+        """
+        Installs the angles phi, theta, and lam into a KAK-type decomposition of the form
+        K(phi) . A(theta) . K(lam) , where K and A are an orthogonal pair drawn from RZGate, RYGate,
+        and RXGate.
+
+        Behavior flags:
+            `simplify` indicates whether gates should be elided / coalesced where possible.
+            `allow_non_canonical` indicates whether we are permitted to reverse the sign of the
+                middle parameter, theta, in the output.  When this and `simplify` are both enabled,
+                we take the opportunity to commute half-rotations in the outer gates past the middle
+                gate, which permits us to coalesce them at the cost of reversing the sign of theta.
+
+        NOTE: The input value of `theta` is expected to lie in [0, pi).
+        """
```

Comment:
Since this is a private method and neither the docs builds or linters care about docstring formatting on this isn't a big issue. But I wonder if it would be better to restructure this in the normal docstrig format for the [napoleon plugin](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) just to be consistent with the rest of the docs. If this were a built and published doc it wouldn't actually pass CI.